verilog/tools/ls/verible-lsp-adapter.h

```diff
@@ -37,7 +38,8 @@ std::vector<verible::lsp::CodeAction> GenerateLinterCodeActions(
 
 // Generate all available code actions.
 std::vector<verible::lsp::CodeAction> GenerateCodeActions(
-    const BufferTracker *tracker, const verible::lsp::CodeActionParams &p);
+    SymbolTableHandler &symbol_table_handler, const BufferTracker *tracker,
```

Comment:
In this project, the preference is to pass things that can be modified as pointer rather than a non-const reference. This improves readability as it is immediately clear that something can be modified.

(exception: `std::ostream` and related as this is somewhat established and expected in all kinds of operator<< situations).